src/Integration.UnitTests/MefServices/VsSessionHostTests.cs

```diff
@@ -371,7 +383,7 @@ public void VsSessionHost_IServiceProvider_GetService()
 
         #region Helpers
 
-        private VsSessionHost CreateTestSubject(ConfigurableActiveSolutionTracker tracker)
+        private VsSessionHost CreateTestSubject(ConfigurableActiveSolutionTracker tracker = null)
```

Comment:
Miscellaneous: line 393 below handles the tracker being `null`, so we might as well make the parameter optional.